### PR TITLE
George/put

### PR DESCRIPTION
**Pull request by palladia**
Make modify/put work with Windows Server, which distinguishes key from non-key attributes.

Modified documentation.

### CODE REVIEW

**Comment by palladia** [comment]
@paulcallen

File: Unix/cli/cli_c.c [hunk 1]
```diff
@@ -222,6 +222,12 @@ static MI_Boolean ArgsToInstance(
         // Get name:
         name = *p++;
 
+        if (name[0] == '*' && name[1] != '\0')
+        {
+            name = &name[1];
+            keyFlag = MI_FALSE;
```
Comment (paulcallen) [inline Unix/cli/cli_c.c]: you will need to reset this flag based on the parameter on every iteration

File: Unix/doc/omi/omi.texi [hunk 2]
```diff
@@ -834,14 +834,14 @@ from the @samp{root/omi} namespace.
...
-have multiple keys. Consider, for example, the following class.
+have multiple keys. Consider, for example, the following class with two keys.
 
 @example
 @verbatim
 class MyClass
 {
     [Key] String  Key1;
-    [Key] Uint32  Key2;
+          Uint32  Key2;
```
Comment (paulcallen) [inline Unix/doc/omi/omi.texi]: A property name of Key implies Key. You may want to pick the third parameter and change the name

**Comment by paulcallen** [comment]
Looks good.

**Pull request closed without merging**